src/types/global.ts

```diff
@@ -21,6 +21,13 @@ export type Config = {
   lang?: LangCodes;
   isFiltersButtonVisible?: boolean;
   maxRecordsPerPage?: number;
+  /**
+   * property for changing behavior of showing tooltip hours
+   * true - will always show 0 taken hours on weekends in day view
```

Comment:
The name of the property suggests that if it's true, the tooltip will show occupancy on weekends as well.

Reply:
Good catch, changed